scripts/js/lib/api/saveImages.ts

```diff
@@ -54,10 +56,20 @@ export async function saveImages(
     if (skipReleaseNote(img.fileName, pkg)) {
       return;
     }
+    const source = `${originalImagesFolderPath}/${img.fileName}`;
+    const dest = `${publicBaseFolder}/${img.dest}`;
 
-    await copyFile(
-      `${originalImagesFolderPath}/${img.fileName}`,
-      `${publicBaseFolder}/${img.dest}`,
-    );
+    if (extname(source) === extname(dest)) {
+      await copyFile(source, dest);
+    } else {
+      try {
+        await $`magick ${source} ${dest}`;
```

Comment:
Does this have output? Should it be quiet?

```suggestion
        await $`magick ${source} ${dest}`.quiet();
```